bin/run_faux

```diff
@@ -2,13 +2,16 @@
 
 ROOT=$(realpath $(dirname $0)/..)
 postfix=
-intf_mac=9a:02:57:1e:8f:00
 docker_args=
 container_prefix=forch-faux
 image=forch/faux
 net_opt=--net=none
 port_num=0
 
+source bin/stack_functions
+set_faux_mac_prefix
+
+intf_mac=$FAUX_MAC_PREFIX:8f:00
```

Comment:
what happens if FAUX_MAC_PREFIX is not defined?  Doesn't this need a default somewhere?

Reply:
set_faux_mac_prefix defines the default